### PR TITLE
[Enhancement] add option to disable disk cache

### PR DESCRIPTION
**Pull request by dirtysalt**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

This PR is to :
- change `block_cache_disk_size` default value to 0
- and it means to disable block cache on disk.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

File: be/src/block_cache/fb_cachelib.cpp [hunk 1]
```diff
@@ -11,22 +11,22 @@ Status FbCacheLib::init(const CacheOptions& options) {
     Cache::Config config;
     config.setCacheSize(options.mem_space_size).setCacheName("default cache").setAccessConfig({25, 10}).validate();
 
-    Cache::NvmCacheConfig nvmConfig;
-    nvmConfig.navyConfig.setBlockSize(4096);
+    if (!options.disk_spaces.empty()) {
```
Comment (GavinMar) [inline be/src/block_cache/fb_cachelib.cpp:14]: Is it necessary to add a new configuration item? As block_cache_disk_size has no default value, if it is not set or is zero, can we treat the disk cache is disabled?

Comment (dirtysalt) [reply be/src/block_cache/fb_cachelib.cpp:14]: OK. I'll fix that.

**Review by GavinMar — DISMISSED** [review]

**Review by GavinMar — APPROVED** [review]

**Review by Youngwb — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Review by caneGuy — APPROVED** [review]
should update the description

**Pull request merged**